documentation/API-design-guidelines.md

```diff
@@ -1976,40 +1976,46 @@ response:
+- **Neither a 3-legged access token nor an explicit User identifier (device or phone number) are provided by the API consumer.**
 
-### Optional device object for 3-legged tokens:
+  One or other MUST be provided to identify the User.
 
-- When using a 3-legged access token, the device associated with the access token must be considered as the device for the API request. This means that the device object is not required in the request, and if included it must identify the same device, therefore **it is recommended NOT to include it in these scenarios** to simplify the API usage and avoid additional validations.
+  In this case, a `422 MISSING_IDENTIFIER` error code MUST be returned, indicating that the API provider cannot identify the User from the provided information.
 
-### Validation mechanism:
+The documentation template below is RECOMMENDED to be used as part of the `info.description` API documentation to explain to the API consumer how the pattern works.
 
-- The server will extract the device identification from the access token, if available.
-- If the API request additionally includes a `device` object when using a 3-legged access token, the API will validate that the device identifier provided matches the one associated with the access token.
-- If there is a mismatch, the API will respond with a 403 - INVALID_TOKEN_CONTEXT error, indicating that the device information in the request does not match the token.
+This template is applicable to CAMARA APIs which:
+- require the User (i.e. resource owner) to be identified; and
+- may have implementations which accept 2-legged access tokens; and
+- do NOT allow the API to confirm whether or not the optional User identifier (`device` or `phoneNumber`) matches that associated with the 3-legged access token
 
-### Error handling for unidentifiable devices:
+The template SHOULD be customised for each API using it by deleting one of the options where marked by (*)
+```md
+# Identifying the [ device | phone number ](*) from the access token
 
-- If the `device` object is not included in the request and the device information cannot be derived from the 3-legged access token, the server will return a 422 `UNIDENTIFIABLE_DEVICE` error.
+This API requires the API consumer to identify a [ device | phone number ](*) as the subject of the API as follows:
+- When the API is invoked using a 2-legged access token, the subject will be identified from the optional [`device` object | `phoneNumber` field](*), which therefore MUST be provided.
+- When a 3-legged access token is used however, this optional identifer MUST NOT be provided, as the subject will be uniquely identified from the access token.
 
-### Restrictions for tokens without an associated authenticated identifier:
+This approach simplifies API usage for API consumers using a 3-legged access token to invoke the API by relying on the information that is associated with the access token and was identified during the authentication process.
```

Comment:
ICM proposes the new text for **Authorization and authentication** in `info.description` in: https://github.com/camaraproject/IdentityAndConsentManagement/pull/214 with the changed writing of  `3-legged` to `three-legged`.
Since both templates go into the same `info.description` we need to harmonize the writing.
I do not have preference - `3-legged` seems to be more often used. 
On the other hand ICM defines [CAMARA-API-access-and-user-consent.md](https://github.com/camaraproject/IdentityAndConsentManagement/blob/main/documentation/CAMARA-API-access-and-user-consent.md)  the term `Three-Legged Access Token `

Reply:
I agree. I should use the ICM defined terms, and they have defined "Three-Legged Access Token" and "Two-Legged Access Token". I'll update the PR accordingly.